lib/engines/content_block_manager/app/services/content_block_manager/get_preview_content.rb

```diff
@@ -4,20 +4,21 @@
 
 module ContentBlockManager
   class GetPreviewContent
-    def initialize(content_id:, content_block_edition:)
-      @content_id = content_id
-      @content_block_edition = content_block_edition
+    def self.for_content_id(content_id:, content_block_edition:)
+      new(content_id:, content_block_edition:).for_content_id
```

Comment:
TIL more about what `self` means in Ruby... had to instantiate a new instance because otherwise in tests the class kept the same context and so the instance variables were also kept 🤔

Reply:
hopefully this makes sense now 😅